docs/datasets/index.rst

```diff
@@ -52,10 +52,28 @@ Note that in Gammapy, 2D image analyses are done with 3D cubes with a single
 energy bin, e.g. for modeling and fitting,
 see the `2D map analysis tutorial <./tutorials/image_analysis.html>`__.
 
-
 To analyse multiple runs, you can either stack the datasets together, or perform
 a joint fit across multiple datasets.
 
+
+Predicted counts
+================
+Predicted counts are computed like:
+
+.. math::
+
+	N_{Pred} = N_{Bkg} + \mathrm{PSF} \circledast \mathrm{EDISP(\mathcal{E} \cdot F_{Src}(l, b, E_{True}))}
+
+
+Where :math:`$N_{Bkg}$` is the expected residual hadronic background,
```

Comment:
isn't the bkg also predicted? It is usually a model. We have a npred_background.